ui/webui/resources/fonts/download_font.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import requests
+import re
+from os import path
+
+font_regex = re.compile(r"\/\* (.*) \*\/\n@font-face {\n\s+font-family: '(.*)';(?:\n|\s|.)+?font-weight: (\d+);(?:\n|\s|.)+?src: url\((.*?)\)(?:.|\n|\s)+?}", re.MULTILINE)
+headers = {
+  'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36',
```

Comment:
I wonder if the server varies response between linux, windows, mac, android given the same browser engine? hope not!

Reply:
Definitely without a `User-Agent` it returns different CSS :laughing: